warehouse/query-core/src/main/java/datawave/query/predicate/EventDataQueryFieldFilter.java

```diff
@@ -47,7 +55,9 @@ public EventDataQueryFieldFilter(ASTJexlScript script, Set<String> nonEventField
             queryFields.add(JexlASTHelper.deconstructIdentifier(identifier));
         }
 
-        setIncludes(queryFields);
+        // Future change, leave in for now.
+        // this.keyProjection = new KeyProjection(queryFields, Projection.ProjectionType.INCLUDES);
+        this.keyProjection = new KeyProjection();
```

Comment:
Should add `this.keyProjection.setIncludes(queryFields)`

Reply:
added in

Reply:
No, ignore me. This constructor path creates a projection but does not set any fields.